clusterctl.yaml.tmpl

```diff
@@ -0,0 +1,24 @@
+CLUSTERCTL_LOG_LEVEL: 10
```

Comment:
since this file is renamed, we need to update makefile prepare-local-clusterctl section which was copying it

Reply:
There is a new step in the docs, please see https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/345#discussion_r1425993109 for an explanation of this change.

Reply:
ok fair enough. ideal would be to not have to read docs :) and automate